modules/geo-layers/src/h3-layers/h3-hexagon-layer.ts

```diff
@@ -99,8 +109,43 @@ const defaultProps = {
 };
 
 // not supported
+// @ts-expect-error
 delete defaultProps.getLineDashArray;
```

Comment:
Just remove. `getLineDashArray` no longer exists on the PolygonLayer defaultProps.